client/src/app.tsx

```diff
@@ -72,7 +72,7 @@ function App() {
                 isGameEnd: updatedRoomConfig.isGameEnd,
                 winningPlayerId: updatedRoomConfig.winningPlayerId,
                 playerNicknameMap: updatedRoomConfig.playerNicknameMap,
-                creatorId: updatedLobbyInfo.createdBy,
+                creatorId: updatedLobbyInfo.createdBy + "",
```

Comment:
This is to force type to be `string`?

Reply:
yea cus remember we made it `| number` a couple weeks ago